x-pack/plugins/infra/server/routes/infra/lib/host/get_hosts_alerts_count.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { kqlQuery, termQuery, termsQuery } from '@kbn/observability-plugin/server';
+import {
+  ALERT_RULE_PRODUCER,
+  ALERT_STATUS,
+  ALERT_STATUS_ACTIVE,
+  ALERT_UUID,
+} from '@kbn/rule-data-utils';
+import { BUCKET_KEY, MAX_SIZE } from '../constants';
+import { InfraAlertsClient } from '../helpers/get_infra_alerts_client';
+
+export type HostAlertsResponse = Array<{
+  name: string;
+  alertsCount: number;
+}>;
+
+export async function getHostsAlertsCount({
+  alertsClient,
+  hostNamesShortList,
+  kuery,
+  from,
+  to,
+  maxNumHosts = MAX_SIZE,
+}: {
+  alertsClient: InfraAlertsClient;
+  hostNamesShortList: string[];
+  kuery?: string;
+  from: string;
+  to: string;
+  maxNumHosts?: number;
+}): Promise<HostAlertsResponse> {
+  const rangeQuery = [
+    {
+      range: {
+        'kibana.alert.time_range': {
+          gte: from,
+          lte: to,
+          format: 'strict_date_optional_time',
+        },
+      },
+    },
+  ];
+  const params = {
+    size: 0,
+    track_total_hits: false,
+    query: {
+      bool: {
+        filter: [
+          ...termQuery(ALERT_RULE_PRODUCER, 'infrastructure'),
```

Comment:
I think we can use the constant `INFRA_ALERT_FEATURE_ID` for `'infrastructure'`

Reply:
I can't use it and I also checked, I couldn't find it used anywhere
<img width="1480" alt="image" src="https://github.com/elastic/kibana/assets/31922082/5cfcc36a-b07b-4e3b-a776-5f4f14b5ba89">

Reply:
@jennypavlova should I move this constant somewhere else? A place where is accessible also for the server?
Also regarding [this comment](https://github.com/elastic/kibana/pull/176034#discussion_r1481791962) from Caue, where do you think is best to create the metric constants? They are used both in the server and the client

Reply:
Hmm true I missed the part that we can't use it in the server and up to you if you want to move it and reuse it. I think using the [constants.ts](https://github.com/elastic/kibana/blob/1dc0076ca870c53555c1bdae5b9f39c99b377109/x-pack/plugins/infra/common/constants.ts) in the common folder should be the place to put the shared constants